includes/free/class-login.php

```diff
@@ -484,6 +484,14 @@ public function process_login() {
             $secure_cookie = is_ssl() ? true : false;
             $user          = wp_signon( apply_filters( 'wpuf_login_credentials', $creds ), $secure_cookie );
 
+            //try with old implementation, which is wrong but we must support that
+            if ( is_wp_error( $user ) ) {
+                $creds['user_login']    = sanitize_text_field( wp_unslash( $_POST['log'] ) );
+                $creds['user_password'] = sanitize_text_field( wp_unslash( $_POST['pwd'] ) );
```

Comment:
I think you also should remove the `sanitize_text_field` from here as it'll change the password.

Reply:
Users who register through our form and use special characters they need to login this way as we stored their password through sanitize_text_field. this is a second try if he fails through trim ( we guess he might register before through this way )

Reply:
Okay